src/components/CurrencyLogo/index.ts

```diff
@@ -0,0 +1,5 @@
+export { CurrencyLogo } from './CurrencyLogo.container'
+export { CurrencyLogoComponent } from './CurrencyLogo.component'
+export { CurrencyWrapperSource } from './CurrencyLogo.types'
+// eslint-disable-next-line
+export type { CurrencyLogoContainerProps } from './CurrencyLogo.types'
```

Comment:
I'm seeing this line too often. I think this fixes the error as seen here: https://github.com/typescript-eslint/typescript-eslint/blob/main/packages/eslint-plugin/docs/rules/consistent-type-exports.md#-correct-1

```suggestion
export { type CurrencyLogoContainerProps } from './CurrencyLogo.types'
```

Reply:
I tried it and unfortunately I was not able to solve it this way even after disabling this rule as suggested in docs you mentioned. There is always some error with prettier/ts. I think we need to upgrade packages related to eslint and then come back to this. Meanwhile I found other solution which involves importing type with alias name and then reexport it again.